docker-storage-setup.sh

```diff
@@ -865,6 +969,16 @@ setup_storage() {
    Fatal "Storage is already configured with ${current_driver} driver. Can't configure it with ${STORAGE_DRIVER} driver. To override, remove $DOCKER_STORAGE and retry."
   fi
 
+  if [ "$STORAGE_DRIVER" == "devicemapper" ] && [ "$DOCKER_ROOT_VOLUME" == "yes" ];then
+   Fatal "STORAGE_DRIVER=${STORAGE_DRIVER} and DOCKER_ROOT_VOLUME=${DOCKER_ROOT_VOLUME} are mutually exclusive options. Use either one of them, not both."
+  fi
+
+  # If docker root is on a separate volume, setup that first.
+  if ! setup_docker_root_lv_fs; then
+    echo "Failed to setup docker root volume."
+    return 1
+  fi
+
```

Comment:
I guess we will have to define in documentation very clearly order of creation of root lv and devicemapper lv. Reason being that they both can pass syntax like 50%FREE. And order of operation will decide which gets what size.

To make sure we don't get surprises, how about creating devicemapper volume first and then root volume.